### DISCUSSION TITLE
openEuler system running error

### DISCUSSION THREAD

**Discussion by clcc2019** [rabbitmq/rabbitmq-server#5140] [Other]
<img width="1143" alt="image" src="https://user-images.githubusercontent.com/46099295/177100767-6df4d6df-d268-4259-a432-bada0bc09b4f.png">
erl_child_setup oom killed

<img width="940" alt="image" src="https://user-images.githubusercontent.com/46099295/177100844-337e3856-f99c-444a-88f1-c1d00e409195.png">

**Comment by michaelklishin** [comment]
We cannot suggest much with this amount of information. No mention of the Erlang, RabbitMQ or OS version used, for example.

According to the screenshots, the process consumed virtually no memory but almost one full CPU core
and that seemingly goes to doing something that fails with a `EBADF` while closing a file descriptor.

**Reply by clcc2019** [reply]
rabbitmq 3.8.9，Is this the problem? https://bugzilla.redhat.com/show_bug.cgi?id=1766224

**Comment by clcc2019** [comment]
<img width="1508" alt="image" src="https://user-images.githubusercontent.com/46099295/177117073-08ffaffd-113c-427d-a788-9e2c8587ccf5.png">
3.10.5
